app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,2 @@
 <resources>
-    <string name="app_name">TrackMyShot</string>
```

Comment:
this doesn't need to be here anymore because we are giving the AppName, back in the grade that we generate.